### DISCUSSION TITLE
Button with dropdown

### DISCUSSION THREAD

**Discussion by VinceG** [saadeghi/daisyui#1813] [Q&A] [answered]

**Comment by saadeghi** [comment, marked as answer]
Currently there's a limitation with [Button group](https://daisyui.com/components/button-group/): It should join the buttons and handle their border radius but currently it can only handles border radius of direct children.  
So it won't be useful if we need a to put a dropdown inside it.  
This will be fixed soon in daisyUI version `3.0`  
But for now, you can use `flex` to put buttons next to each other and change the border radius of buttons accordingly:  

https://play.tailwindcss.com/rY9gYnOzcz

```
<div class="flex items-center">
  <button class="btn rounded-r-none">Button</button>

  <div class="dropdown dropdown-end">
    <label tabindex="0" class="btn btn-square my-1 rounded-l-none">▼</label>
    <ul tabindex="0" class="dropdown-content menu p-2 bg-base-200 rounded-box w-52">
      <li><a>Item 1</a></li>
      <li><a>Item 2</a></li>
    </ul>
  </div>

</div>
```
